dune/gdt/spaces/rt/dune-pdelab-wrapper.hh

```diff
@@ -66,6 +66,7 @@ public:
   static_assert(polOrder == 0, "Untested!");
   static_assert(rangeDim == GridLayerType::dimension, "Untested!");
   static_assert(rangeDimCols == 1, "Untested!");
+  static const constexpr Backends backend_type{Backends::pdelab};
```

Comment:
s.a.

Reply:
What I meant (and wrote at another place above, at least I thought so) was: I would prefer sticking to
```c++
static const constexpr Backends backend_type = Backends::pdelab;
```
which I find more readable somehow, if you have no objections

Reply:
I don't mind if you change that personally, but it's actually good practice to use it.